<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InflowParameterWidget</class>
 <widget class="QFrame" name="InflowParameterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>967</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Turbulent Inflow Model Paraemeters</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>638</width>
        <height>941</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="typeSelectionGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Profile type</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="selectUniform">
            <property name="text">
             <string>uniform</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="selectExponential">
            <property name="text">
             <string>exponential</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>reset to default values</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="phiTensorGroup">
         <property name="title">
          <string>Coefficient tensor PHI</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="3" column="0">
           <widget class="QDoubleSpinBox" name="PHI21"/>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="PHI22"/>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="PHI12">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QDoubleSpinBox" name="PHI33"/>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="PHI32"/>
          </item>
          <item row="6" column="0">
           <widget class="QDoubleSpinBox" name="PHI31"/>
          </item>
          <item row="2" column="0">
           <widget class="QDoubleSpinBox" name="PHI11"/>
          </item>
          <item row="3" column="2">
           <widget class="QDoubleSpinBox" name="PHI23">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="PHI13">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="alphaParameterGroup">
         <property name="title">
          <string>Power law: alpha coefficients</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="alpha3"/>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="alpha1"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="alpha2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="referencePointGroup">
         <property name="title">
          <string>Inflow coordinate system</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="3" column="2">
           <widget class="QDoubleSpinBox" name="zVector3"/>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="zVector2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>orientation 1 (n-vector)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="z02"/>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="z03"/>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="z01"/>
          </item>
          <item row="3" column="0">
           <widget class="QDoubleSpinBox" name="zVector1"/>
          </item>
          <item row="5" column="2">
           <widget class="QDoubleSpinBox" name="tVector3"/>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="tVector2"/>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QPushButton" name="btnNormalize">
            <property name="text">
             <string>normalize</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>orientation 2 (t-vector)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QDoubleSpinBox" name="tVector1"/>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>reference point</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="lengthScaleGroup">
         <property name="title">
          <string>Integral length scales</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="Lwz"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_flowdir">
            <property name="text">
             <string>flow dir</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="Lvx"/>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="Lvy"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_perpendicular">
            <property name="text">
             <string>perpendicular</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="Lwx"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="Lux"/>
          </item>
          <item row="2" column="3">
           <widget class="QDoubleSpinBox" name="Lvz"/>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="Luy"/>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="Luz"/>
          </item>
          <item row="3" column="2">
           <widget class="QDoubleSpinBox" name="Lwy"/>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_up_dir">
            <property name="text">
             <string>vertical</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_u_component">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>u</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_v_component">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>v</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_w_component">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>w</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
